channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -99,7 +112,7 @@ class SlackChannel private constructor(
 
     companion object : JaicpCompatibleAsyncChannelFactory {
         override val channelType = "slack"
-        override fun create(botApi: BotApi, apiUrl: String) = SlackChannel(botApi, apiUrl)
+        override fun create(botApi: BotApi, apiUrl: String) = SlackChannel(botApi, "$apiUrl/")
```

Comment:
I think we shouldn't just put a slash at the end and make a user think more about how URL should look like.
Let's just do something like `apiUrl.takeIf { ie.endsWith("/") } ?: apiUrl + "/"`

Reply:
@nikvoloshin we have `String.toUrl()` utility function that fixes all double slashes in the URL

Reply:
@morfeusys Thanks for pointing that out! 
Then it should be called inside constructor: `methodsEndpointUrlPrefix = urlPrefix.toUrl()`.